features/kuzzle.feature

```diff
@@ -351,6 +353,7 @@ Feature: Kuzzle functional tests
   @realtime
   Scenario: get list of subscriptions
     Given A room subscription listening to "lastName" having value "Hopper"
+    And I wait 0.1s
```

Comment:
(nitpicking) wouldn't this latency be better located in the subscription task above?

Reply:
I am not sure I get it. 
The cluster latency is between the time the subscription is sent and the time it is replicated to be able to compute the list.

Reply:
What I'm saying is that you're waiting 0.1s every time you call `Given A room subscription listening...`, so I thought it might be best to wait that 0.1s in the code of that "given" task, instead of in the features files.

Reply:
ah ok. I am not sure about that. I would prefer to keep it explicit in the scenario that we are adding a small replication delay rather than hiding it in the subscription step.